src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/UpdateInductionTest.kt

```diff
@@ -258,50 +259,298 @@ class UpdateInductionTest : IntegrationTestBase() {
       .hasReasonToNotGetWork(emptySet())
       .wasModifiedBy("auser_gen")
       .wasCreatedAt(createdDateTime)
-      .wasLastModifiedAfter(initialModifiedDateTime)
+      .wasLastModifiedAfter(initialModifiedAt)
     assertThat(updatedInduction.workExperience).isEquivalentTo(expectedWorkExperience)
     assertThat(updatedInduction.skillsAndInterests).isEquivalentTo(expectedSkillsAndInterests)
     assertThat(updatedInduction.qualificationsAndTraining).isEquivalentTo(expectedQualificationsAndTraining)
     assertThat(updatedInduction.inPrisonInterests).isEquivalentTo(expectedInPrisonInterests)
     // check last modified dates of child objects
-    assertThat(updatedInduction.workExperience!!.modifiedDateTime).isAfter(initialModifiedDateTime)
-    assertThat(updatedInduction.skillsAndInterests!!.modifiedDateTime).isAfter(initialModifiedDateTime)
-    assertThat(updatedInduction.qualificationsAndTraining!!.modifiedDateTime).isAfter(initialModifiedDateTime)
-    assertThat(updatedInduction.inPrisonInterests!!.modifiedDateTime).isAfter(initialModifiedDateTime)
+    assertThat(updatedInduction.workExperience!!.modifiedDateTime).isAfter(initialModifiedAt)
+    assertThat(updatedInduction.skillsAndInterests!!.modifiedDateTime).isAfter(initialModifiedAt)
+    assertThat(updatedInduction.qualificationsAndTraining!!.modifiedDateTime).isAfter(initialModifiedAt)
+    assertThat(updatedInduction.inPrisonInterests!!.modifiedDateTime).isAfter(initialModifiedAt)
+  }
+
+  @Test
+  fun `should update induction with no qualifications`() {
```

Comment:
These new tests cover what I think is currently possible to remove from an Induction, starting with qualifications (but not previous training, since that is mandatory, even if it only contains `NONE`).